### ISSUE TITLE
Enter/Return behaviour

### ISSUE THREAD

**Issue by JulianLeviston** [atom/language-clojure#23]
When I press newline, the cursor should drop to the same indentation level, but it seems to go to the start of the line preceding it. How would I change this behaviour? Or is it not really about the language-clojure package?

**Comment by WalterGR** [comment]
Note that this also applies to brackets, not just parens.

If I type

    (loop [result []

and press Enter/Return, I expect the new line to be indented and my cursor positioned as follows:

    (loop [result []
           _

but it positions the cursor at the beginning of the new line.

**Comment by GetContented** [comment]
Yes. All brackets, rounded (parens), square, curly (braces) etc.
